experimental/uefi/app/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "Apache-2.0"
 uefi = { version = "*", features = ["exts"] }
 uefi-services = "*"
 log = { version = "*" }
+oak_remote_attestation = { path = "../../../remote_attestation/rust" }
```

Comment:
I am surprised this does not cause cargo udeps to complain, seeing that the code does not currently use anything from the remote attestation crate. Do we ignore the UEFI app when running cargo udeps?

Reply:
Good catch, forgot about us using `cargo udeps`. The reason it doesn't complain is that we run it on the workspace, and the UEFI app is excluded from the workspace bc of https://github.com/project-oak/oak/issues/2654.

Should we keep this as is? This crate should start actually using this dep fairly soon anyways.

Reply:
I think it is fine for now, but we should probably update xtask at some point to run cargo udeps for this as well.